ddtrace/tracer/span.go

```diff
@@ -200,6 +200,7 @@ func (s *span) setMeta(key, v string) {
 	if s.Meta == nil {
 		s.Meta = make(map[string]string, 1)
 	}
+	delete(s.Metrics, key)
```

Comment:
I'm not sure why these are still showing up from the other PR. A rebase of the most recent changes onto `v1` and/or a new branch might help.